legal-api/src/legal_api/reports/report_v2.py

```diff
@@ -148,12 +148,23 @@ def _format_furnishing_data(self):
             processed_date = LegislationDatetime.as_legislation_timezone(self._report_date_time)
         self._report_data['furnishing']['processedDate'] = processed_date.strftime(OUTPUT_DATE_FORMAT)
 
+        self._report_data['furnishing']['businessLegalType'] = 'BC'
+        if self._business.legal_type in [Business.LegalTypes.EXTRA_PRO_A.value]:
+            self._report_data['furnishing']['businessLegalType'] = 'EP'
```

Comment:
I think we should keep using `A` as the legal type value for EPs.
Do we actually have a need distinguish BC companies in the letter templates?

If not, I think we can replace all 3 lines with
i.e `self._report_data['furnishing']['businessLegalType'] =  self._business.legal_type`

From a quick look at `legal-api/report-templates/noticeOfDissolutionCommencement.html`, I think all you need is ` if furnishing.businessLegalType == 'A'`

Reply:
Updated